src/workflows-app/SubmissionConfig.test.js

```diff
@@ -1,5 +1,3 @@
-import '@testing-library/jest-dom';
```

Comment:
`@testing-library/jest-dom` is imported in test setup and does not need to be repeated in individual test files.

Reply:
Handled in #4063, hasn't been merged in yet

Reply:
Since this PR removes `@testing-library/jest-dom` from Terra UI's dependencies, it must also remove these imports which can no longer be resolved.

It's fine if multiple PRs make the same change, they won't conflict.